theme.json

```diff
@@ -0,0 +1,413 @@
+{
+  "$schema": "https://schemas.wp.org/trunk/theme.json",
+  "version": 2,
+  "settings": {
+    "appearanceTools": true,
+    "useRootPaddingAwareAlignments": true,
+    "border": {
+      "color": false,
+      "radius": false,
+      "style": false,
+      "width": false
+    },
+    "color": {
+      "text": false,
+      "background": false,
+      "link": false,
+      "custom": false,
+      "defaultPalette": false,
+      "defaultGradients": false,
+      "defaultDuotone": false,
+      "customGradient": false,
+      "customDuotone": false,
+      "palette": [
+        {
+          "slug": "text",
+          "color": "var(--wp--custom--text--secondary)",
+          "name": "text"
+        },
+        {
+          "slug": "heading",
+          "color": "var(--wp--custom--text--primary)",
+          "name": "heading"
+        },
+        {
+          "slug": "background",
+          "color": "var(--wp--custom--background--primary)",
+          "name": "background"
+        }
+      ],
+      "gradients": []
+    },
+    "custom": {
+      "body": {
```

Comment:
You can probably get away with two body styles. Where you might run into issues is with helper text and other bits of UI text like labels. I think that's where we created a support token set for type styles that were to support inputs. A complete starter should probably include those as tokens even if their aliasing the body type styles since they have such a different use in the application.

Reply:
That's def fair, and I understand why more would be needed. I'm thinking that for our purpose we leave it at 2, and note that more can be added if needed. We can provide examples for why more might be needed, noting down your reasoning for it.